lib/services/auth_service.dart

```diff
@@ -117,12 +110,11 @@ class AuthService {
         case AuthorizationErrorCode.canceled:
           break;
         default:
-          yield AddProblem(
-            problem: AppleSignInProblem.by(
-              (b) => b
-                ..message = e.toString()
-                ..info = BuiltMap<String, String>({'code': e.code}),
-            ),
+          yield createAddProblem(
+            AppleSignInProblem,
+            e,
```

Comment:
Do you think we should change `e` to `error`?  I *think* we tend to use `error` and it'd be good to be consistent.  I'm aware the old code uses `e` but given we're making a change anyway, what do you think?